SPECS/python-rich/python-rich.spec

```diff
@@ -10,7 +10,12 @@ Source0:        %{url}/archive/v%{version}/rich-%{version}.tar.gz
 
 BuildArch:      noarch
 
-Patch0:          3229.patch
+Patch0:         3229.patch
+Patch1:         ptest-warning.patch
```

Comment:
Could you add more comments about this patch? Either here or in the patch itself?

Reply:
Sure I will add a comment to the spec

Reply:
added.